README.md

```diff
@@ -173,26 +184,25 @@ export BYOC=true
 make ocm/aws/create_access_key
 ```
 
-3. Create cluster template: `make ocm/cluster.json`.
+4. Create cluster template: `make ocm/cluster.json`.
 
 This command will generate `ocm/cluster.json` file with generated cluster name. This file will be used as a template to create your cluster via OCM CLI.
 By default, it will set the expiration timestamp for a cluster for 4 hours, meaning your cluster will be automatically deleted after 4 hours after you generated this template. If you want to change the default timestamp, you can update it in `ocm/cluster.json` or delete the whole line from the file if you don't want your cluster to be deleted automatically at all. 
 
-4. Create the cluster: `make ocm/cluster/create`.
+5. Create the cluster: `make ocm/cluster/create`.
 
 This command will send a request to [Red Hat OpenShift Cluster Manager](https://cloud.redhat.com/) to spin up your cluster and waits until it's ready. You can see the details of your cluster in `ocm/cluster-details.json` file
 
-5. Once your cluster is ready, OpenShift Console URL will be printed out together with the `kubeadmin` user & password. These are also saved to `ocm/cluster-credentials.json` file. Also there will be `ocm/cluster.kubeconfig` file created that you can use for running `oc` commands right away, for example, for listing all projects on your OpenShift cluster:
+6. Once your cluster is ready, OpenShift Console URL will be printed out together with the `kubeadmin` user & password. These are also saved to `ocm/cluster-credentials.json` file. Also there will be `ocm/cluster.kubeconfig` file created that you can use for running `oc` commands right away, for example, for listing all projects on your OpenShift cluster:
 
 ```
 oc --config ocm/cluster.kubeconfig projects
 ```
 
-6. If you want to install the latest released RHMI, you can trigger it by applying an RHMI addon.
+7. If you want to install the latest released RHMI, you can trigger it by applying an RHMI addon.
 Run `make ocm/install/rhmi-addon` to trigger the installation. Once the installation is completed, the installation CR with RHMI components info will be printed to the console.
 
-7. If you want to delete your cluster, run `make ocm/cluster/delete`
-**BYOC**
+8. If you want to delete your cluster, run `make ocm/cluster/delete`
```

Comment:
LGTM, but maybe we should mention that for clusters using AWS stuff `rhmi` resource should be deleted first?

Reply:
I have actually included it in the [make target](https://github.com/pawelpaszki/integreatly-operator/pull/1/files#diff-c90e3921bdbf84a8bb9bb3e7179702e5R92), so it will delete the CR every time when the target is called.
I can also add a condition to the target to not delete the CR if it's not BYOC, but I thought deleting it each time will not cause any harm. Wdyt?

Reply:
That sounds like a good approach. We will see how it goes I guess.